plugins/woocommerce-blocks/assets/js/blocks/cart/inner-blocks/cart-order-summary-subtotal/frontend.tsx

```diff
@@ -1,6 +1,13 @@
 /**
  * Internal dependencies
  */
-import Block from './block';
+import Block, { BlockAttributes } from './block';
+import { DEFAULT_HEADING } from './constants';
 
-export default Block;
+const Frontend = ( { className = '', heading }: BlockAttributes ) => {
+	const headingElement = heading ?? DEFAULT_HEADING;
```

Comment:
@nielslange this was intentional, I was pretty cognizant of why I was using `??` here because I set the block json values to `null` as default. intentionally this allows displaying no value. I see no reason not to allow that here but the placeholder means you can always find your way back to the initial state

Reply:
> Could you elaborate on why a merchant would want to have one or more headings intentionally left empty on the frontend?

It's inconsistent because the headers have different purposes. but one place i could imagine it, for example the "add a coupon" menu, you may want to remove the text, and then adding an icon to open it with custom style and the text would no longer intrude on your design.

> We currently ensure this consistency with the Proceed to Checkout and the Place Order button label.

One thing that was tripping me up is we allow empty values for the `CART TOTALS` heading. So we're obviously not too consistent across the codebase. But those that you mention are more recent and were done intentionally, so yeah maybe we should stick to that approach across the board. You did actually mention that to me in our review session and I forgot about it.

> Allowing merchants to leave labels empty could introduce a new behavior that might lead to confusion, especially if a merchant inadvertently overlooks an empty heading on the frontend due to seeing the placeholder in the page editor.

Yes I see your point about the placeholder. because it's subtle they won't maybe notice it's empty by accident. That is a compelling reason not to allow it. I'm torn on this one so maybe it's just safer not to allow it as you say. 👍🏻